website/content/en/configurations/identity-access-modes/workload-identity-mode.md

```diff
@@ -21,7 +21,6 @@ spec:
   provider: azure
   parameters:
     usePodIdentity: "false"         # set to true for pod identity access mode
-    clientID: "<client id of the Azure AD Application or user-assigned managed identity to use for workload identity>"
```

Comment:
To keep backward compatibility should we keep this parameter and make it optional? Customers who onboarded may need to keep their services working.

Reply:
Hi @herquan-docusign, honestly I would assume that it breaks no working application as you require to have the client id configured on the service account for the driver to work even in the current versions. https://learn.microsoft.com/en-us/azure/aks/csi-secrets-store-identity-access#configure-workload-identity

Reply:
So for existing setups, it would just ignore the client id in the SPC parameters